app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   has_many :videos, through: :user_videos
 
   validates :email, uniqueness: true, presence: true
-  validates :password, presence: true
+  validates :password, on: :create, presence: true
```

Comment:
the reason i changed this was because when i went to update the user, it was saying that password couldn't be blank however since were not updating the password, we don't need to include that. Thoughts are that we should include update as well as the user may update their password and we need to ensure they are putting in a password